packages/migrate/src/utils/ensureDatabaseExists.ts

```diff
@@ -7,7 +7,7 @@ import prompt from 'prompts'
 import type execa from 'execa'
 
 export type MigrateAction = 'create' | 'apply' | 'unapply' | 'dev' | 'push'
-export type DbType = 'MySQL' | 'PostgreSQL' | 'SQLite' | 'SQL Server'
+export type DbType = 'MySQL' | 'PostgreSQL' | 'SQLite' | 'SQL Server' | 'CockroachDB'
```

Comment:
Unrelated: we don't need MongoDB here?

Reply:
Since `db push` works with MongoDB, indeed, I'll have a look later at this!